### PR TITLE
Document `-f` alias for framework option in list package command

### LINKED ISSUE #40340: Document -f alias for framework option in list package command
### Type of issue

Other (describe below)

### Description

New feature in .NET 9 - short form of --framework option, -f.

### Page URL

https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-list-package

### Content source URL

https://github.com/dotnet/docs/blob/main/docs/core/tools/dotnet-list-package.md

### Document Version Independent Id

3d4b052c-33fa-cc83-4df6-4a3341f12763

### Article author

@tdykstra

### Metadata

* ID: 8400fd6a-d987-1056-283f-9364fbce3c86 
* Service: **dotnet-fundamentals**

### PR DESCRIPTION
**Pull request by ghost**
Fixes #40340 

Following dotnet/sdk#37448, this PR documents the new `-f` alias for the framework option in list package command.

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/tools/dotnet-list-package.md](https://github.com/dotnet/docs/blob/ed4906dcd8a6842667add78f61f63ed331bf93dd/docs/core/tools/dotnet-list-package.md) | [dotnet list package](https://review.learn.microsoft.com/en-us/dotnet/core/tools/dotnet-list-package?branch=pr-en-us-38759) |

### CODE REVIEW

**Review by tdykstra — COMMENTED** [review]
This option isn't available yet. Which SDK version will it be in?

**Comment by joegoldman2** [comment]
> This option isn't available yet. Which SDK version will it be in?

I would say .NET 9 but I'm not sure. Maybe @baronfel or @marcpopMSFT can help to reply.

**Comment by marcpopMSFT** [comment]
The change went into main so it'll ship in 9.0.100 in the fall.

File: docs/core/tools/dotnet-list-package.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ The project or solution file to operate on. If not specified, the command search
 
   Displays packages that have been deprecated.
   
-- **`--framework <FRAMEWORK>`**
+- **`-f|--framework <FRAMEWORK>`**
 
   Displays only the packages applicable for the specified [target framework](../../standard/frameworks.md). To specify multiple frameworks, repeat the option multiple times. For example: `--framework net6.0 --framework netstandard2.0`.
```
Comment (tdykstra) [inline docs/core/tools/dotnet-list-package.md:87]:
```suggestion
  Displays only the packages applicable for the specified [target framework](../../standard/frameworks.md). To specify multiple frameworks, repeat the option multiple times. For example: `--framework net6.0 --framework netstandard2.0`. The short form of the option (`-f`) is available starting in .NET 9 SDK.
```

**Review by tdykstra — APPROVED** [review]
Thanks! I left a suggestion to note that the short form of the option is available in .NET 9, and added a do-not-merge label to ensure that the PR doesn't get merged until the first .NET 9 preview that includes it is released.

**Review by tdykstra — APPROVED** [review]
Thanks, @joegoldman2 !

**Comment by tdykstra** [comment]
This is included in Preview 2 so I'll merge it now. The markdownlint errors are not for this PR.

**Pull request merged**